### DISCUSSION TITLE
Multiple expects

### DISCUSSION THREAD

**Discussion by oldshaghat** [vitest-dev/vscode#278] [Q&A] [answered]
I don't know if I've missed something in how to set things up or interpret what I have, but I have a test case with multiple expect statements in it and when the test fails, I wish the extension would highlight the specific expect that caused the failure. I just get a banner at the top of the test with "expected true to be false" or whatever and the entire test is highlighted. 
I know I can add a descriptive line to the expect call but I guess I assumed it would know which one it was - but I can't see how to get it to tell me. Am I just doing it wrong? i.e I should adopt a more JS testing style here, have one expect per test or something?

**Comment by sheremet-va** [comment, marked as answer]
This is a bug. The error should be shown on every failed expect
